### PR TITLE
Apartment ID Fix

### PR DESCRIPTION
**Pull request by ab7289**
Updates our URLs, templates, and tests to use the apartment primary key instead of suite_num.

### CODE REVIEW

File: location/templates/apartment.html [hunk 1]
```diff
@@ -3,7 +3,13 @@
 {% block breadcrumbs %}
 <a href="{% url 'search' %}">Search</a>
 <a href="{% url 'location' pk=apt.location.id %}">{{ apt.location.full_address }}</a>
-<span>{{ apt.suite_num }}</span>
+<span>
+  {% if apt.suite_num %}
+  {{ apt.suite_num }}
+  {% else %}
+  1
```
Comment (alldne) [inline location/templates/apartment.html]: I think showing `1` could be a bit misleading. How about showing other placeholder, for instance `-`, `N/A`, `Unknown suite number`?

Comment (ab7289) [reply location/templates/apartment.html]: what about a `#`? Unknown suite number seems like it indicates that something broke, so I don't particularly love that option. A `-` might be tricky to click also.

Comment (alldne) [reply location/templates/apartment.html]: +1 for `-` is tricky to click lol. `#` is also fine for me

File: location/templates/claim.html [hunk 2]
```diff
@@ -3,7 +3,13 @@
 {% block breadcrumbs %}
 <a href="{% url 'search' %}">Search</a>
 <a href="{% url 'location' pk=apt.location.id %}">{{ apt.location.full_address }}</a>
-<a href="{% url 'apartment' pk=apt.location.id suite_num=apt.suite_num %}">{{ apt.suite_num }}</a>
+<a href="{% url 'apartment' pk=apt.location.id apk=apt.id %}">
+    {% if apt.suite_num %}
+    {{ apt.suite_num }}
+    {% else %}
+    1
```
Comment (alldne) [inline location/templates/claim.html]: This should be also changed to `#`?

File: location/templates/claim.html [hunk 3]
```diff
@@ -24,13 +30,21 @@
 {% block content %}
 <h1>Claim</h1>
 <p>
-    Are you a tenant or a landlord of <a href="{% url 'apartment' pk=apt.location.id suite_num=apt.suite_num %}">#{{ apt.suite_num }}</a>
+    Are you a tenant or a landlord of <a href="{% url 'apartment' pk=apt.location.id apk=apt.id %}">
+        {% if apt.suite_num %}
+        #{{ apt.suite_num }}
+        {% else %}
+        this apartment
```
Comment (alldne) [inline location/templates/claim.html]: Nice

File: location/templates/claim.html (hunk 2 above)
Comment (ab7289) [reply location/templates/claim.html]: good catch, I felt like I was missing it somewhere.

**Pull request merged**